src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -298,12 +298,23 @@ def main():
     orig_model_path = args.model_path
     args.model_path = model_to_path(args.model_path)
 
+    if args.num_streams:
+        _LOGGER.info("num_streams set to {}".format(args.num_streams))
+    elif not args.num_streams and scenario not in "singlestream":
+        # If num_streams isn't defined, find a default
+        args.num_streams = max(1, int(args.num_cores / 2))
```

Comment:
is the /2 because of hyper threads?

Reply:
That's the original code. I just moved it thinking I was going to minimize the change size. But that's probably not right. I think the right answer is None, here. I'm going to change it.

Reply:
It's changed. BAM!

Reply:
@bnellnm @Willtor this num_streams is used to generate the number of python threads submitting inferences to the engine, so it cannot be None it needs to be a real number